priv/repo/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 10.1
--- Dumped by pg_dump version 10.1
+-- Dumped from database version 10.0
+-- Dumped by pg_dump version 10.0
```

Comment:
Can we see if there's a way to avoid the db version numbering while still getting the benefits of the structure.sql?

Reply:
I'm not seeing any documented way of doing so with `mix ecto.dump`